### PR TITLE
updated get-player-inventory and execute-command

### PR DESCRIPTION
**Pull request by matt-bot1**
added auth requirements to get-player-inventory
added auth requirements to execute-commands

### CODE REVIEW

File: machines/teleport-player.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            type: 'string',
+            description: 'Authorization token to send with the request',
+            example: "EOGHZANOIZEAHZFUR93573298539242F3NG",
+            required: true,
+            whereToGet: {
+                description: 'Set in webpermission.xml or with webtokens telnet command'
+            }
+        },
+
+        playerID: {
```
Comment (niekcandaele) [inline machines/teleport-player.js]: Input name should conform to camelCase

(name should be playerId instead of playerID)

Comment (niekcandaele) [reply machines/teleport-player.js]: Also, i think entity ID OR steam ID will work, maybe this should be updated in description?

File: machines/teleport-player.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+    fn: function(inputs, exits) {
+        const _ = require("lodash")
+        var executeCommand = require('machine').build(require('./execute-command.js'))
+     console.log()
+	 
+        let reason = ''
+        if (!_.isUndefined(inputs.reason)) {
+            reason = inputs.reason
+        }
```
Comment (niekcandaele) [inline machines/teleport-player.js]: Is reason still an input? This is left over from copying kick-player machine I think

File: machines/teleport-player.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        }).exec({
+            success: function(response) {
+	    console.log(response)
+                if (response.result == '*** ERROR: Executing command \'tele\' failed could not find player') {
+                    return exits.unknownPlayer("Could not find player")
+		console.log(response)
+                }
+                if (_.startsWith(response.result, "teleporting player")) {
+                    return exits.success(response);
+		console.log(response)
```
Comment (niekcandaele) [inline machines/teleport-player.js]: Some unneeded console.log() lines left over from developing

File: machines/teleport-player.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            type: 'string',
+            description: 'Steam ID of the player to teleport',
+            example: '76561198028175841',
+            required: true
+        },
+
+        coordinates: {
+            type: 'string',
+            description: 'co ordinates to send a player',
+			required: true,
```
Comment (niekcandaele) [inline machines/teleport-player.js]: Take care of whitespacing please

File: execute-command.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+      extendedDescription: "Make sure this is the port for the web server, not telnet or ...",
+      required: true,
+      example: "8082",
+    },
+
+    authName: {
+      type: 'string',
+      description: 'Authorization name to send with the request',
+      example: "csmm",
+	  required: true,
```
Comment (niekcandaele) [inline execute-command.js]: Whitespace

**Review by niekcandaele — CHANGES_REQUESTED** [review]
Good stuff, just needs some minor adjustments here and there

File: teleport-player.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+        var executeCommand = require('machine').build(require('./execute-command.js'))
+	 
+        executeCommand({
+            ip: inputs.ip,
+            port: inputs.port,
+            authName: inputs.authName,
+            authToken: inputs.authToken,
+            command: `tele ${inputs.playerId} ${inputs.coordinates}`
+        }).exec({
+            success: function(response) {
```
Comment (matt-bot1) [inline teleport-player.js]: a little lost with this bit response.result== if you could help

**Review by niekcandaele — APPROVED** [review]

**Pull request merged**